Index: git/cpuproc_analyzer.cpp
===================================================================
--- git.orig/cpuproc_analyzer.cpp
+++ git/cpuproc_analyzer.cpp
@@ -36,7 +36,7 @@
 #include <stdarg.h>
 
 #ifdef PROCANALYZER_BROADBAND
-#include <telemetry_busmessage_sender.h>
+//#include <telemetry_busmessage_sender.h>
 #include <json-c/json.h>
 #endif
 
@@ -1415,7 +1415,7 @@ int LogProcData(stProcData* procData, in
                       FDCount, procData->ld_num_threads-1);
              RDK_LOG(RDK_LOG_INFO, "LOG.RDK.CPUPROCANALYZER","Event : %s, Value : %s\n", eventName,
                      telemetry_buf);
-             t2_event_s(eventName, telemetry_buf);
+             //t2_event_s(eventName, telemetry_buf);
          }
        #endif
     }
@@ -1742,7 +1742,7 @@ int main(int argc, char** argv)
         pEnvConfig = ENV_ACTUAL_PATH;
 
     #if defined  PROCANALYZER_BROADBAND
-    t2_init("cpuprocanalyzer");
+    //t2_init("cpuprocanalyzer");
     #endif
 
     char tmp_string[256]={0};
Index: git/Makefile.am
===================================================================
--- git.orig/Makefile.am
+++ git/Makefile.am
@@ -29,7 +29,7 @@ else
 AM_LDFLAGS = -lpthread -lrdkloggers 
 
 if PROCANALYZER_BROADBAND
-AM_LDFLAGS += -ltelemetry_msgsender -ljson-c
+AM_LDFLAGS +=  -ljson-c
 AM_CXXFLAGS = -DPROCANALYZER_BROADBAND
 endif
 endif
