Index: git/source/lm/lm_main.c
===================================================================
--- git.orig/source/lm/lm_main.c
+++ git/source/lm/lm_main.c
@@ -94,7 +94,7 @@
 #define OnboardLog(...)
 #endif
 
-#include <telemetry_busmessage_sender.h>
+//#include <telemetry_busmessage_sender.h>
 #define TELEMETRY_MAX_BUFFER 256
 
 #define LM_IPC_SUPPORT
@@ -615,7 +615,7 @@ static void LM_SET_ACTIVE_STATE_TIME_(in
 				{
 				CcspTraceWarning(("RDKB_CONNECTED_CLIENTS: Wifi client with %s MacAddress and %s HostName gone offline\n",pHost->pStringParaValue[LM_HOST_PhysAddressId],pHost->pStringParaValue[LM_HOST_HostNameId]));
 				OnboardLog("RDKB_CONNECTED_CLIENTS: Wifi client with %s MacAddress and %s HostName gone offline\n",pHost->pStringParaValue[LM_HOST_PhysAddressId],pHost->pStringParaValue[LM_HOST_HostNameId]);
-				t2_event_d("WIFI_INFO_clientdisconnect", 1);
+				//t2_event_d("WIFI_INFO_clientdisconnect", 1);
 				}
 #ifndef USE_NOTIFY_COMPONENT
 				remove_Mac_to_band_mapping(pHost->pStringParaValue[LM_HOST_PhysAddressId]);
@@ -686,7 +686,7 @@ static void LM_SET_ACTIVE_STATE_TIME_(in
                     if(!lm_wrap_checkIPv4AddressInRange(lan_ip_address, pHost->pStringParaValue[LM_HOST_IPAddressId], lan_net_mask))
                     {
                         CcspTraceWarning(("<%s> IPAddress out of range : IPAddress = %s, MAC Addr = %s \n",__FUNCTION__, pHost->pStringParaValue[LM_HOST_IPAddressId], pHost->pStringParaValue[LM_HOST_PhysAddressId]));
-                        t2_event_d("SYS_ERROR_IPAOR", 1);
+                        //t2_event_d("SYS_ERROR_IPAOR", 1);
                     }
                 }
                 else {
@@ -790,7 +790,7 @@ static void LM_SET_ACTIVE_STATE_TIME_(in
 					get_uptime(&uptime);
                   			CcspTraceWarning(("Client_Connect_complete:%d\n",uptime));	
 					OnboardLog("Client_Connect_complete:%d\n",uptime);
-					t2_event_d("btime_clientconn_split", uptime);
+					//t2_event_d("btime_clientconn_split", uptime);
 				   }
 					CcspTraceWarning(("RDKB_CONNECTED_CLIENTS: Client type is %s, MacAddress is %s and HostName is %s Connected  \n",interface,pHost->pStringParaValue[LM_HOST_PhysAddressId],pHost->pStringParaValue[LM_HOST_HostNameId]));
 					lmHosts.lastActivity++;
@@ -2681,13 +2681,13 @@ static void *Hosts_LoggingThread(void *a
 #endif
 		CcspTraceWarning(("-------------------------------------------------------------------\n"));
 
-		t2_event_d("Total_devices_connected_split", TotalDevCount);
-		t2_event_d("Total_online_clients_split", TotalOnlineDev);
-		t2_event_d("Total_offline_clients_split", TotalOffLineDev);
-		t2_event_d("Total_wifi_clients_split", TotalWiFiDev);
-		t2_event_d("Total_Ethernet_Clients_split", TotalEthDev);
+	//	t2_event_d("Total_devices_connected_split", TotalDevCount);
+	//	t2_event_d("Total_online_clients_split", TotalOnlineDev);
+	//	t2_event_d("Total_offline_clients_split", TotalOffLineDev);
+	//	t2_event_d("Total_wifi_clients_split", TotalWiFiDev);
+	//	t2_event_d("Total_Ethernet_Clients_split", TotalEthDev);
 #if !defined (NO_MOCA_FEATURE_SUPPORT)
-		t2_event_d("Total_MoCA_Clients_split", TotalMoCADev);
+	//	t2_event_d("Total_MoCA_Clients_split", TotalMoCADev);
 #endif
 	
 		/* CID 340337 Unused value fix */
@@ -3423,18 +3423,6 @@ void Wifi_ServerSyncHost (char *phyAddr,
 																	       (pHost->pStringParaValue[LM_HOST_HostNameId]) ? (pHost->pStringParaValue[LM_HOST_HostNameId]) : "NULL"));	
 					memset(telemetryBuff, 0, TELEMETRY_MAX_BUFFER);
 					snprintf(telemetryBuff,16,"%s",pHost->pStringParaValue[LM_HOST_HostNameId]);
-					if(strncmp(telemetryBuff,"SC",strlen("SC")) == 0)
-					{
-						t2_event_d("WIFI_INFO_XHCAM_online", 1);
-					}
-					else if(strncmp(telemetryBuff,"android",strlen("android")) == 0)
-					{
-						t2_event_d("WIFI_INFO_XHTS_online", 1);
-					}
-					else
-					{
-						t2_event_d("WIFI_INFO_XHclient_online", 1);
-					}
 				}
 			}  
 			else 
@@ -3445,18 +3433,6 @@ void Wifi_ServerSyncHost (char *phyAddr,
 
 					memset(telemetryBuff, 0, TELEMETRY_MAX_BUFFER);
 					snprintf(telemetryBuff,16,"%s",pHost->pStringParaValue[LM_HOST_HostNameId]);
-					if(strncmp(telemetryBuff,"SC",strlen("SC")) == 0)
-					{
-						t2_event_d("WIFI_INFO_XHCAM_offline", 1);
-					}
-					else if(strncmp(telemetryBuff,"android",strlen("android")) == 0) 
-					{
-						t2_event_d("WIFI_INFO_XHTS_offline", 1);
-					}
-					else
-					{
-						t2_event_d("WIFI_INFO_XHclient_offline", 1);
-					}
 				}
 			}
 
Index: git/source/lm/lm_wrapper.c
===================================================================
--- git.orig/source/lm/lm_wrapper.c
+++ git/source/lm/lm_wrapper.c
@@ -59,7 +59,7 @@
 #include "safec_lib_common.h"
 
 // TELEMETRY 2.0 //RDKB-25996
-#include <telemetry_busmessage_sender.h>
+//#include <telemetry_busmessage_sender.h>
 
 /* Fix RDKB-499 */
 #define DHCPV4_RESERVED_FORMAT  "%17[^,],%63[^,],%63[^,]"
@@ -1649,7 +1649,7 @@ void lm_wrapper_get_dhcpv4_client()
                     ERR_CHK(rc);
                     lmHosts.lastActivity++;
                     CcspTraceWarning(("Hostname Changed <%s> <%d> : Hostname = %s HostVersionID %lu\n",__FUNCTION__, __LINE__,pHost->pStringParaValue[LM_HOST_HostNameId],lmHosts.lastActivity));
-		    t2_event_d("SYS_INFO_Hostname_changed", 1);
+		    //t2_event_d("SYS_INFO_Hostname_changed", 1);
                     if (syscfg_set_u_commit(NULL, "X_RDKCENTRAL-COM_HostVersionId", lmHosts.lastActivity) != 0)
                     {
                         AnscTraceWarning(("syscfg_set failed\n"));
@@ -1768,7 +1768,7 @@ void lm_wrapper_get_dhcpv4_reserved()
 					ERR_CHK(rc);
 					lmHosts.lastActivity++;
 					CcspTraceWarning(("Hostname Changed <%s> <%d> : Hostname = %s HostVersionID %lu\n",__FUNCTION__, __LINE__,pHost->pStringParaValue[LM_HOST_HostNameId],lmHosts.lastActivity));
-					t2_event_d("SYS_INFO_Hostname_changed", 1);
+					//t2_event_d("SYS_INFO_Hostname_changed", 1);
 					if (syscfg_set_u_commit(NULL, "X_RDKCENTRAL-COM_HostVersionId", lmHosts.lastActivity) != 0)
 					{
 						AnscTraceWarning(("syscfg_set failed\n"));
Index: git/source/Ssp/ssp_main.c
===================================================================
--- git.orig/source/Ssp/ssp_main.c
+++ git/source/Ssp/ssp_main.c
@@ -38,7 +38,7 @@
 #endif
 #endif
 
-#include "telemetry_busmessage_sender.h"
+//#include "telemetry_busmessage_sender.h"
 #include "lm_main.h"
 #include <sys/stat.h>
 #include "ssp_global.h"
@@ -47,7 +47,7 @@
 #include "cap.h"
 
 #ifdef INCLUDE_BREAKPAD
-#include "breakpad_wrapper.h"
+//#include "breakpad_wrapper.h"
 #endif
 
 #if defined(_ENABLE_EPON_SUPPORT_)
@@ -304,10 +304,10 @@ int main(int argc, char* argv[])
     }
     pComponentName          = CCSP_COMPONENT_NAME_LMLITE;
 #ifdef INCLUDE_BREAKPAD
-    breakpad_ExceptionHandler();
+    //breakpad_ExceptionHandler();
 #endif
 
-    t2_init("ccsp-lm-lite");
+    //t2_init("ccsp-lm-lite");
 
     if ( bRunAsDaemon ) 
         daemonize();
Index: git/source/lm/Makefile.am
===================================================================
--- git.orig/source/lm/Makefile.am
+++ git/source/lm/Makefile.am
@@ -50,7 +50,7 @@ libCcspLM_la_SOURCES += cosa_xhosts_dml.
                         cosa_managementserver_dml.c
 endif
 
-libCcspLM_la_LDFLAGS = -lhal_platform -lccsp_common -lsyscfg -lsysevent -lutapi -lutctx -lulog -ltelemetry_msgsender -lavro -lmsgpackc -ltrower-base64 -luuid -lm -lrt -pthread -lnanomsg -lwrp-c -llibparodus -lsecure_wrapper
+libCcspLM_la_LDFLAGS = -lhal_platform -lccsp_common -lsyscfg -lsysevent -lutapi -lutctx -lulog  -lavro -lmsgpackc -ltrower-base64 -luuid -lm -lrt -pthread -lnanomsg -lwrp-c -llibparodus -lsecure_wrapper
 if ENABLE_SESHAT
 libCcspLM_la_LDFLAGS += -llibseshat
 endif
