diff --git a/source/dml/wifi_ssp/ssp_main.c b/source/dml/wifi_ssp/ssp_main.c
index 9e0c808b..80ff2fbc 100644
--- a/source/dml/wifi_ssp/ssp_main.c
+++ b/source/dml/wifi_ssp/ssp_main.c
@@ -605,7 +605,7 @@ wifi_util_dbg_print(WIFI_MGR,"%s:%d: RDK_LOGGER_INIT done!\n", __func__, __LINE_
 
 #endif
   
-    t2_init("ccsp-wifi-agent");
+   // t2_init("ccsp-wifi-agent");
   
     /* Default handling of SIGCHLD signals */
     if (signal(SIGCHLD, SIG_DFL) == SIG_ERR)
diff --git a/source/stubs/wifi_stubs.c b/source/stubs/wifi_stubs.c
index a489538f..77de99ef 100644
--- a/source/stubs/wifi_stubs.c
+++ b/source/stubs/wifi_stubs.c
@@ -84,7 +84,6 @@ static int strcpy_func(char *dst, size_t max, const char *src)
 
 wifi_stubs_descriptor_t stubs_desc = {
     t2_event_d,
-    t2_event_s,
     v_secure_system,
     drop_root,
     gain_root_privilege,

