Index: git/source/dml/wifi_ssp/ssp_main.c
===================================================================
--- git.orig/source/dml/wifi_ssp/ssp_main.c
+++ git/source/dml/wifi_ssp/ssp_main.c
@@ -44,7 +44,7 @@
 #include "ccsp_dm_api.h"
 #include "ccsp_WifiLog_wrapper.h"
 #include "syscfg/syscfg.h"
-#include "telemetry_busmessage_sender.h"
+//#include "telemetry_busmessage_sender.h"
 #include "secure_wrapper.h"
 
 #if defined (FEATURE_SUPPORT_WEBCONFIG)
@@ -605,7 +605,7 @@ wifi_util_dbg_print(WIFI_MGR,"%s:%d: RDK
 
 #endif
   
-    t2_init("ccsp-wifi-agent");
+    //t2_init("ccsp-wifi-agent");
   
     /* Default handling of SIGCHLD signals */
     if (signal(SIGCHLD, SIG_DFL) == SIG_ERR)
Index: git/source/stubs/wifi_stubs.c
===================================================================
--- git.orig/source/stubs/wifi_stubs.c
+++ git/source/stubs/wifi_stubs.c
@@ -25,11 +25,12 @@
 #include <string.h>
 #include "wifi_hal.h"
 
-#ifdef ONEWIFI_RDKB_APP_SUPPORT
+//#ifdef ONEWIFI_RDKB_APP_SUPPORT
+#ifdef 0
 #include "safec_lib_common.h"
 
-extern int t2_event_d(char *marker, int value);
-extern int t2_event_s(char *marker, char *buff);
+int t2_event_d(char *marker, int value);
+int t2_event_s(char *marker, char *buff);
 extern int v_secure_system(const char *command);
 extern bool drop_root();
 extern void gain_root_privilege();
Index: git/source/core/wifi_ctrl.c
===================================================================
--- git.orig/source/core/wifi_ctrl.c
+++ git/source/core/wifi_ctrl.c
@@ -21,7 +21,7 @@
 #include <stdbool.h>
 #include "wifi_stubs.h"
 #include "wifi_hal.h"
-#include "wifi_hal_rdk_framework.h"
+//#include "wifi_hal_rdk_framework.h"
 #include "wifi_ctrl.h"
 #include "wifi_mgr.h"
 #include "wifi_util.h"
Index: git/source/core/wifi_ctrl_queue_handlers.c
===================================================================
--- git.orig/source/core/wifi_ctrl_queue_handlers.c
+++ git/source/core/wifi_ctrl_queue_handlers.c
@@ -28,7 +28,7 @@
 #include "scheduler.h"
 #include <unistd.h>
 #include <pthread.h>
-#include "wifi_hal_rdk_framework.h"
+//#include "wifi_hal_rdk_framework.h"
 #include "wifi_passpoint.h"
 #include "wifi_stubs.h"
 
